### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by lintool**
+ Update README
+ Update onboarding docs
+ Add deprecation message to Solrini and Elastrini

### CODE REVIEW

File: docs/experiments-msmarco-passage.md [hunk 1]
```diff
@@ -300,6 +300,9 @@ At this time, look back through the learning outcomes again and make sure you're
 As a next step in the onboarding path, you basically [do the same thing again in Python with Pyserini](https://github.com/castorini/pyserini/blob/master/docs/experiments-msmarco-passage.md) (as opposed to Java with Anserini here).
 
 Before you move on, however, add an entry in the "Reproduction Log" at the bottom of this page, following the same format: use `yyyy-mm-dd`, make sure you're using a commit id that's on the main trunk of Anserini, and use its 7-hexadecimal prefix for the link anchor text.
+In the text of your pull request, please provide some details on your setup (for example, operating system, environment and configuration, etc.).
```
Comment (sahel-sh) [inline docs/experiments-msmarco-passage.md:303]:
```suggestion
In the description of your pull request, please provide some details on your setup (for example, operating system, environment and configuration, etc.).
```

Comment (sahel-sh) [inline docs/experiments-msmarco-passage.md:303]:
```suggestion
In the text of your pull request, please provide some details on your setup (e.g., operating system, environment and configuration, etc.).
```

File: docs/solrini.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # Solrini: Anserini Integration with Solr
 
+⁉️ **Important Note:** As part of Anserini's upgrade to Lucene 9, support for Solrini was removed at [`272565`](https://github.com/castorini/anserini/commit/27256551e958f39495b04e89ef55de9d27f33414) on August 2, 2022.
+The features documented below no longer work, and this guide is retained only for historical reasons.
```
Comment (sahel-sh) [inline docs/solrini.md:4]:
```suggestion
The features documented below are no longer available, and this guide is retained only for historical reasons.
```

File: docs/experiments-cord19-extras.md [hunk 3]
```diff
@@ -1,5 +1,8 @@
 # Ingesting CORD-19 into Solr and Elasticsearch
 
+⁉️ **Important Note:** As part of Anserini's upgrade to Lucene 9, support for Solrini was removed at [`272565`](https://github.com/castorini/anserini/commit/27256551e958f39495b04e89ef55de9d27f33414) on August 2, 2022.
+The features documented below no longer work, and this guide is retained only for historical reasons.
```
Comment (sahel-sh) [inline docs/experiments-cord19-extras.md:4]:
```suggestion
The features documented below are no longer available, and this guide is retained only for historical reasons.
```

File: docs/start-here.md [hunk 4]
```diff
@@ -275,8 +275,12 @@ From here, you're now ready to proceed to try and reproduce the [BM25 Baselines
 ](experiments-msmarco-passage.md).
 
 Before you move on, however, add an entry in the "Reproduction Log" at the bottom of this page, following the same format: use `yyyy-mm-dd`, make sure you're using a commit id that's on the main trunk of Anserini, and use its 7-hexadecimal prefix for the link anchor text.
+In the text of your pull request, please provide some details on your setup (for example, operating system, environment and configuration, etc.).
```
Comment (sahel-sh) [inline docs/start-here.md:278]:
```suggestion
In the description of your pull request, please provide some details on your setup (e.g., operating system, environment and configuration, etc.).
```

File: docs/start-here.md [hunk 5]
```diff
@@ -275,8 +275,12 @@ From here, you're now ready to proceed to try and reproduce the [BM25 Baselines
...
+In addition, also provide some indication of success (e.g., everything worked) or document issues you encountered.
+If you think this guide can be improved in any way (e.g., you caught a typo or think a clarification is warranted), feel free to include in the pull request.
```
Comment (sahel-sh) [inline docs/start-here.md:280]:
```suggestion
If you think this guide can be improved in any way (e.g., you caught a typo or think a clarification is warranted), feel free to include it in the pull request.
```

File: docs/experiments-msmarco-passage.md [hunk 6]
```diff
@@ -300,6 +300,9 @@ At this time, look back through the learning outcomes again and make sure you're
...
+In addition, also provide some indication of success (e.g., everything worked) or document issues you encountered.
+If you think this guide can be improved in any way (e.g., you caught a typo or think a clarification is warranted), feel free to include in the pull request.
```
Comment (sahel-sh) [inline docs/experiments-msmarco-passage.md:305]:
```suggestion
If you think this guide can be improved in any way (e.g., you caught a typo or think a clarification is warranted), feel free to include it in the pull request.
```

File: README.md [hunk 7]
```diff
@@ -316,10 +303,14 @@ In turn, you'll be recognized as a [contributor](https://github.com/castorini/an
 
 Beyond that, there are always [open issues](https://github.com/castorini/anserini/issues) we would appreciate help on!
 
-## ℹ️ Release History
+## 📜️ Release History
 
 + v0.21.0: March 31, 2023 [[Release Notes](docs/release-notes/release-notes-v0.21.0.md)]
 + v0.20.0: January 20, 2023 [[Release Notes](docs/release-notes/release-notes-v0.20.0.md)]
```
[the remark this answers is not in the document]
Comment (lintool) [reply README.md:309]: Ah, it was a jump to sync up with the versioning in pyserini.

File: README.md [hunk 8]
```diff
@@ -297,13 +290,7 @@ For the most part, manual copying and pasting of commands into a shell is requir
 + Runbooks for TREC 2018: [[Anserini group](docs/runbook-trec2018-anserini.md)] [[h2oloo group](docs/runbook-trec2018-h2oloo.md)]
 + Runbook for [ECIR 2019 paper on axiomatic semantic term matching](docs/runbook-ecir2019-axiomatic.md)
 + Runbook for [ECIR 2019 paper on cross-collection relevance feedback](docs/runbook-ecir2019-ccrf.md)
-
-### Other Features
-
```
[the remark this answers is not in the document]
Comment (lintool) [reply README.md:302]: I explicitly wanted more compact bullets.

**Comment by lintool** [comment]
@sahel-sh thanks for the detailed look. CR addressed.

**Review by sahel-sh — APPROVED** [review]

**Pull request merged**